src/Components/GrowthChart/GrowthChartBuilder/GrowthChartBuilder.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Line } from 'react-chartjs-2';
+import 'chart.js/auto';
+
+
+interface ChartData {
+    datasets: {
+        [key: string]: {
+            [key: string]: number;
+        }[];
+    };
+    metadata: {
+        label: string;
+        value: string;
+        yaxis: string;
+        xaxis: string;
+    }[];
+}
+
+interface GrowthChartBuilderProps {
+    chartData?: ChartData;
+}
+
+export const GrowthChartBuilder = ({ chartData }: GrowthChartBuilderProps) => {
+    const dataSetData = chartData.datasets.Girls0to5Years
+    const dataSetMetadata = chartData.metadata[0]
+    const TimeInterval = Object.keys(dataSetData);
+
+    const Keys = Object.keys(dataSetData[0]);
+    const percentileKeys = Keys.slice(1, Keys.length);
```

Comment:
idk if i like this, because objects does not really have an order. It will probably be in this order each time, thats why i just mention it. Imagine if SD1 is first, then this will remove that column